packages/datagrid/src/components/dataset-serializer/utils.test.js

```diff
@@ -0,0 +1,193 @@
+import { getColumnDefs, getRowData, getPinnedColumnDefs, getValueByCell } from './utils';
+
+const sample = {
+	schema: {
+		type: 'record',
+		name: 'StringArrayRecord',
+		fields: [
+			{
+				name: 'field0',
+				doc: 'Nom de la gare',
+				type: {
+					type: 'string',
+					dqType: 'FR Commune',
```

Comment:
Can we prefix that type with something else than `dq`? It doesn't really make sense to prefix it with a product name.